main.sh

```diff
@@ -266,19 +242,11 @@ for test in "$@"; do
     prepare_test_log_dir $test
     case $test in
         IOR)
-            #start_server
-            #start_agent
-	    #create_pool
-            #run_IOR
-            #killall_proc $test
-	    echo "WIP to enable IOR on daos master. Please use older commit for running IOR on 1.0.0"
-            ;;
-        DAOS_TEST)
-            #start_server
-            #start_agent
-            #run_daos_test
-            #killall_proc $test
-	    echo "DAOS_TEST is temporarily disabled - TBD"
+            start_server
+            start_agent
+	    create_pool
+            run_IOR
```

Comment:
Add destroy_pool after running IOR

Reply:
I will leave that to you in future patch since it doesn't affect the ior results in the use case here.